honeybee_radiance/primitive/material/glass.py

```diff
@@ -0,0 +1,299 @@
+            modifier=modifier,
+            dependencies=dependencies)
+        # update the values from values dictionary.
+        # this might look redundant but it is to ensure any strange input like inputs in
+        # strings line are updated and will be the same as input in cases that user uses
+        # from_string classmethod
+        cls_.values = values_dict['values']
+        return cls_
+
+    @classmethod
+    def from_dict(cls, input_dict):
+        """Make radiance glass from a dictionary.
+        {
+            "name": "", // Material Name
+            "type": "glass",
+            "r_transmissivity": float, // Transmissivity for red
+            "g_transmissivity": float, // Transmissivity for green
+            "b_transmissivity": float, // Transmissivity for blue
+            "refraction_index": float, // Index of refraction
+            "modifier": "", // material modifier (Default: "void")
+            "dependencies": []
+        }
+        """
+        assert 'type' in input_dict, 'Input dictionary is missing "type".'
+        if input_dict['type'] != 'glass':
+            raise ValueError('Type must be glass not %s.' % input_dict['type'])
+
+        modifier, dependencies = Material.filter_dict_input(input_dict)
+
+        return cls(name=input_dict["name"],
+                   r_transmissivity=input_dict["r_transmissivity"],
+                   g_transmissivity=input_dict["g_transmissivity"],
+                   b_transmissivity=input_dict["b_transmissivity"],
+                   refraction_index=input_dict["refraction_index"],
+                   modifier=modifier,
+                   dependencies=dependencies)
+
+    def to_dict(self):
+        """Translate radiance material to json
+        {
+            "type": "glass", // Material type
+            "name": "", // Material Name
+            "r_transmissivity": float, // Transmissivity for red
+            "g_transmissivity": float, // Transmissivity for green
+            "b_transmissivity": float, // Transmissivity for blue
+            "refraction_index": float, // Index of refraction
+            "modifier": "", // material modifier (Default: "void")
+            "dependencies": []
+        }
+        """
+        return {
+            "modifier": self.modifier.to_dict(),
+            "type": "glass",
+            "name": self.name,
+            "r_transmissivity": self.r_transmissivity,
+            "g_transmissivity": self.g_transmissivity,
+            "b_transmissivity": self.b_transmissivity,
+            "refraction_index": self.refraction_index,
+            "dependencies": [dp.to_dict() for dp in self.dependencies]
+        }
+
+    @staticmethod
+    def get_transmissivity(transmittance):
+        """Calculate transmissivity based on transmittance value.
+
+        "Transmissivity is the amount of light not absorbed in one traversal of
+        the material. Transmissivity -- the value usually measured - is the total
```

Comment:
I believe that you mean to state `Transmittance -- the value usually measured`.  Otherwise, this docstring doesn't make much sense.